deploy/deploy.sh

```diff
@@ -1,57 +1,107 @@
 #!/bin/bash
 
-# Script that deploys the latest Collabosphere code from
-# a specified remote and branch. The `DOCUMENT_ROOT` environment
-# variable should be set to the directory in which the static
-# need to be deployed
-#
-#  usage: $ deploy/deploy.sh
-
 # Fail the entire script when one of the commands in it fails
 set -e
 
-# Get the remote and branch that should be
-# deployed from the provided environment variables
-TARGET_REMOTE=${REMOTE:-origin}
-TARGET_BRANCH=${BRANCH:-master}
+# Go to base directory of local git repo
+cd $(dirname "${BASH_SOURCE[0]}")/..
+base_directory="${PWD}"
+
+echo_usage() {
+  echo; echo "USAGE"; echo "  ${0} [-r remote] [-b branch] [-t tag]"; echo
+  echo "Deploy the latest SuiteC code from a specified branch or tag."; echo
+  echo "Set environment variable DOCUMENT_ROOT to the Apache directory which will serve SuiteC static files."; echo
+  echo "Common usages:"; echo
+  echo "   # Deploy Arthur's branch"
+  echo "   ${0} -r arthur_guinness -b irish_dry_stout"; echo
+  echo "   # Deploy qa branch using default remote (origin)"
+  echo "   ${0} -b qa"; echo
+  echo "   # Deploy tag 1.6"
+  echo "   ${0} -t 1.6"; echo
```

Comment:
@paulkerschen Usage info is fixed. The examples give a sense of what is optional and what is required.